src/components/LinkButton.tsx

```diff
@@ -0,0 +1,29 @@
+/* eslint-disable jsx-a11y/anchor-is-valid */
+import { Icon, Link, LinkText } from 'design-system-react';
+import type { ComponentPropsWithoutRef, ReactNode } from 'react';
+
+interface LinkButtonProperties {
+  icon: string;
+  children: ReactNode;
+  onClick: React.ChangeEvent;
+}
+
+function LinkButton({
+  className,
+  icon = 'plus',
+  onClick,
+  children,
+}: ComponentPropsWithoutRef<'a'> & LinkButtonProperties): JSX.Element {
+  return (
+    <Link
```

Comment:
This component looks good, but this needs to be a `<button>` not a `<a>` in the html: generally links take you places (either other pages or somewhere else on the same page) while buttons perform actions (like adding another institution to this form). 

We want to use the right semantic html (button), otherwise it's going to introduce a bunch of weird accessibility issues. For example if you test out this form now, since there's no href for this link, it's not added to the tab order of the page. So if I'm using keyboard navigation on this form, I can't navigate to this action.